basic_samples/SDS/Java/readme.rst

```diff
@@ -386,28 +386,28 @@ or when the properties have the same name, Sds will map the properties automatic
         jsonMultipleValues = sdsclient.getRangeValues(tenantId, namespaceId, sampleStream.getId(), "1", 0, 3, false, SdsBoundaryType.ExactOrCalculated, sampleViewId);
```

Comment:
sampleViewId is now sampleStreamViewId in the code so looks like it should be updated here too to match the code